kolibri/plugins/learn/assets/src/vue/search-page/index.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+
+  <div>
+    <form @submit.prevent="submitSearch">
```

Comment:
Instead of a 'submit' style search, I would expect this to be simply trigger a debounced/throttled search on keypress.

good reference: http://ux.stackexchange.com/questions/34360/delay-on-keystroke-when-search-as-you-type/34363#34363